modules/systemd-journal/journal-reader.c

```diff
@@ -598,7 +598,7 @@ _init(LogPipe *s)
   if (!log_source_init(s))
     return FALSE;
 
-  gint res = journald_open(self->journal, SD_JOURNAL_LOCAL_ONLY);
+  gint res = journald_open(self->journal, SD_JOURNAL_LOCAL_ONLY | SD_JOURNAL_ALL_NAMESPACES);
```

Comment:
Passing this flag should be configurable, as this could be a not desired behaviour. Making this as a default is okay.

Reply:
agreed

Reply:
When possible to do either runtime and build time configuration, in this project the runtime is one to go.
The suggestion was to introduce a config file option either setting it or not.

This future of systemd introduces two option, one with this flag and one with the specifying the namespace. 

```
systemd-journal( namespace('foo') );

systemd-jouranl( namespace('') ); # invalid
systemd-journal( all-namespaces( yes/ no ) );

systemd-journal( namespace('bar') all-namespaces( yes ) ); #invalid
```
In the last case it could be said that the `all-namespaces` are stronger, but imho it is better simply not accepting so user could make informed decision.

Of course it is fine, if this PR now only focuses on the `all-namespaces(yes/no)` and not implement the namespaces.

Reply:
I've tried to tackle the all-namespaces option. Somewhere must be an error. No matter if I set yes or no, all namespaces are always included.

The namespace() option i won't do, because i think this requires bigger changes.